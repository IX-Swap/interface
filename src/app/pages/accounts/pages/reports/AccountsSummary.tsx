import React from 'react'
import { Grid } from '@material-ui/core'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { VSpacer } from 'components/VSpacer'
import { ReportsInfo } from 'app/pages/accounts/pages/reports/components/ReportsInfo/ReportsInfo'
import { Actions } from './components/Actions/Actions'
import { ReportsAccordion } from 'app/pages/accounts/pages/reports/components/ReportsAccordion/ReportsAccordion'
import { useGetActivitySummary } from 'app/pages/accounts/hooks/useGetActivitySummary'

export const AccountsSummary: React.FC = () => {
  const { data, isLoading } = useGetActivitySummary()
  console.log(data, isLoading)

  return (
    <Grid container direction={'column'}>
      <Grid item>
        <PageHeader title='Activity Summary' />
        <VSpacer size={'medium'} />
      </Grid>

      <ReportsInfo>
        <Actions onContractClick={() => null} onExpandClick={() => null} />
      </ReportsInfo>

      <Grid item>
        <VSpacer size={'medium'} />

        <ReportsAccordion summary={'Open Positions'}>
          <div>Open Positions</div>
        </ReportsAccordion>

        <ReportsAccordion summary={'Cash Report'}>
          <div>Cash Report</div>
        </ReportsAccordion>
      </Grid>
    </Grid>
  )
}
