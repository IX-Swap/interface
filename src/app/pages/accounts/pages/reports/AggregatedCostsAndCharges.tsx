import React, { useEffect } from 'react'
import { Grid } from '@material-ui/core'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { VSpacer } from 'components/VSpacer'
import { ReportsInfo } from 'app/pages/accounts/pages/reports/components/ReportsInfo/ReportsInfo'
import { ReportsAccordion } from 'app/pages/accounts/pages/reports/components/ReportsAccordion/ReportsAccordion'
import { useFeeAndCharges } from 'app/pages/accounts/hooks/useFeeAndCharges'
import { FeesTable } from 'app/pages/accounts/pages/reports/components/FeesTable/FeesTable'
import { LoadingIndicator } from 'app/components/LoadingIndicator/LoadingIndicator'
import { NoData } from 'app/pages/accounts/pages/reports/components/NoData/NoData'
import { useQueryFilter } from 'hooks/filters/useQueryFilter'

export const AggregatedCostsAndCharges: React.FC = () => {
  const { data, isLoading } = useFeeAndCharges()
  const { updateFilter } = useQueryFilter()

  useEffect(() => {
    updateFilter('expandedSections', 'Fees (SGD),Fees (USD)')
  }, [])

  const hasSGDData = !!data?.sgd?.length
  const hasUSDData = !!data?.usd?.length

  const renderContent = () => {
    if (!hasSGDData && !hasUSDData) {
      return <NoData />
    }
    if (hasSGDData) {
      return (
        <ReportsAccordion summary={'Fees (SGD)'}>
          <FeesTable accounts={data.sgd} total={data.totalSgd} />
        </ReportsAccordion>
      )
    }

    if (hasUSDData) {
      return (
        <ReportsAccordion summary={'Fees (USD)'}>
          <FeesTable accounts={data.usd} total={data.totalUsd} />
        </ReportsAccordion>
      )
    }
  }

  if (isLoading) {
    return <LoadingIndicator />
  }

  return (
    <Grid container direction={'column'}>
      <Grid item>
        <PageHeader title='Aggregated Costs and Charges' />
        <VSpacer size={'medium'} />
      </Grid>

      <ReportsInfo />

      <Grid item>
        <VSpacer size={'medium'} />
        {renderContent()}
      </Grid>
    </Grid>
  )
}
