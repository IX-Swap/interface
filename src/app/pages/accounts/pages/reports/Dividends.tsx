import React, { useEffect } from 'react'
import { Grid } from '@mui/material'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { VSpacer } from 'components/VSpacer'
import { ReportsInfo } from 'app/pages/accounts/pages/reports/components/ReportsInfo/ReportsInfo'
import { ReportsAccordion } from 'app/pages/accounts/pages/reports/components/ReportsAccordion/ReportsAccordion'
import { useDividends } from 'app/pages/accounts/hooks/useDividends'
import { DividendsTable } from 'app/pages/accounts/pages/reports/components/DividendsTable/DividendsTable'
import { LoadingIndicator } from 'app/components/LoadingIndicator/LoadingIndicator'
import { NoData } from 'app/pages/accounts/pages/reports/components/NoData/NoData'
import { useQueryFilter } from 'hooks/filters/useQueryFilter'

export const Dividends: React.FC = () => {
  const { updateFilter } = useQueryFilter()
  const { data, isLoading } = useDividends()

  useEffect(() => {
    updateFilter('expandedSections', 'Dividends')
  }, [])

  const renderContent = () => {
    if (data === undefined || data.length < 1) {
      return <NoData />
    }
    return (
      <ReportsAccordion summary={'Dividends'}>
        <DividendsTable data={data} />
      </ReportsAccordion>
    )
  }

  if (isLoading) {
    return <LoadingIndicator />
  }

  return (
    <Grid container direction={'column'}>
      <Grid item>
        <PageHeader title='Dividends' />
        <VSpacer size={'medium'} />
      </Grid>

      <ReportsInfo />

      <Grid item>
        <VSpacer size={'medium'} />
        {renderContent()}
      </Grid>
    </Grid>
  )
}
