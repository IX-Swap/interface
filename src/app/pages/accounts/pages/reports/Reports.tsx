import React from 'react'
import { Box, Grid, Typography } from '@mui/material'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { VSpacer } from 'components/VSpacer'
import { Filters } from './components/Filters/Filters'
import { ReportsTable } from 'app/pages/accounts/pages/reports/components/ReportsTable/ReportsTable'

export const Reports: React.FC = () => {
  return (
    <Grid container direction={'column'}>
      <Grid item>
        <PageHeader title='Reports' />
      </Grid>
      <Grid item>
        <VSpacer size={'small'} />
        <Typography variant={'h4'}>Statements</Typography>
        <VSpacer size={'small'} />
      </Grid>
      <Grid
        item
        container
        justifyContent={'space-between'}
        alignItems={'center'}
      >
        <Grid item>
          <Box marginRight={2} marginTop={2} marginBottom={2}>
            <Typography variant={'body1'}>
              View, manage and track your accounting reports and taxations.
            </Typography>
          </Box>
        </Grid>
        <Grid item>
          <Filters />
        </Grid>
      </Grid>
      <Grid item>
        <VSpacer size={'medium'} />
        <ReportsTable />
      </Grid>
    </Grid>
  )
}
