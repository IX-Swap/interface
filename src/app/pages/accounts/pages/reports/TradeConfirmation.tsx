import React from 'react'
import { Grid } from '@material-ui/core'
import { PageHeader } from 'app/components/PageHeader/PageHeader'
import { VSpacer } from 'components/VSpacer'
import { ReportsInfo } from 'app/pages/accounts/pages/reports/components/ReportsInfo/ReportsInfo'
import { ReportsAccordion } from 'app/pages/accounts/pages/reports/components/ReportsAccordion/ReportsAccordion'
import { useGetExchangeFills } from 'app/pages/accounts/hooks/useGetExchangeFills'

export const TradeConfirmation: React.FC = () => {
  const { data, isLoading } = useGetExchangeFills()
  console.log(data, isLoading)

  return (
    <Grid container direction={'column'}>
      <Grid item>
        <PageHeader title='Trade Confirmation' />
        <VSpacer size={'medium'} />
      </Grid>

      <ReportsInfo />

      <Grid item>
        <VSpacer size={'medium'} />

        <ReportsAccordion summary={'Trades'}>
          <div>Trades</div>
        </ReportsAccordion>
      </Grid>
    </Grid>
  )
}
