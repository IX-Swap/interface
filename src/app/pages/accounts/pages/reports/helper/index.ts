import { formatAmount } from 'helpers/numbers'

export const formatValue = (value: string | number) =>
  typeof value === 'number' ? formatAmount(value) : value
