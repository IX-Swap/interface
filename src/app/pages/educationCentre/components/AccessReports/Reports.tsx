import { Grid } from '@mui/material'
import { columns } from 'app/pages/educationCentre/components/AccessReports/columns'
import { ReportTypeFilter } from 'app/pages/educationCentre/components/AccessReports/ReportTypeFilter'
import { TableView } from 'components/TableWithPagination/TableView'
import { homeURL } from 'config/apiURL'
import { homeQueryKeys } from 'config/queryKeys'
import { useQueryFilter } from 'hooks/filters/useQueryFilter'
import React from 'react'

export const Reports = () => {
  const { getFilterValue } = useQueryFilter()

  return (
    <Grid container spacing={3}>
      <Grid item xs={12}>
        <ReportTypeFilter />
      </Grid>
      <Grid item xs={12}>
        <TableView
          name={homeQueryKeys.getAtlasOneAccessReports}
          uri={homeURL.getAtlasOneAccessReports}
          paperProps={{
            variant: 'elevation',
            elevation: 0
          }}
          filter={{
            reportType: getFilterValue('reportType')
          }}
          themeVariant='primary'
          noHeader
          bordered={false}
          columns={columns}
        />
      </Grid>
    </Grid>
  )
}
