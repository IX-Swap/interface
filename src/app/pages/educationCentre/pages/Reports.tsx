import React, { Fragment } from 'react'
import { Box, Grid, Typography, useTheme } from '@material-ui/core'
import { RootContainer } from 'ui/RootContainer'
import { Reports as ReportsComponent } from 'app/pages/educationCentre/components/AccessReports/Reports'
import AtlasLogoLight from 'assets/icons/atlas_logo_white.png'
import AtlasLogoDark from 'assets/icons/atlas_logo.png'
import { VSpacer } from 'components/VSpacer'

export const Reports = () => {
  const theme = useTheme()

  return (
    <RootContainer>
      <VSpacer size={'small'} />
      <Fragment>
        <Grid container direction='column' spacing={10}>
          <Grid item xs={12}>
            <Grid item container spacing={1} alignItems='center' wrap='nowrap'>
              <Typography variant='h2'>Reports</Typography>
              <Typography
                variant='body2'
                style={{
                  lineHeight: '100%',
                  marginRight: theme.spacing(1),
                  marginLeft: theme.spacing(2)
                }}
              >
                In Partnership With
              </Typography>
              <img
                width={106}
                height={34}
                src={
                  theme.palette.type === 'light'
                    ? AtlasLogoDark
                    : AtlasLogoLight
                }
                alt={'Atlas One Logo'}
              />
            </Grid>
            <Box my={2.5} />
            <ReportsComponent />
          </Grid>
        </Grid>
      </Fragment>
    </RootContainer>
  )
}
