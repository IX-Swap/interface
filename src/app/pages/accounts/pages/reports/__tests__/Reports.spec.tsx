import React from 'react'
import { render } from 'test-utils'
import { Reports } from 'app/pages/accounts/pages/reports/Reports'

describe('Reports', () => {
  afterEach(async () => {})

  it('should match snapshot', () => {
    const { container } = render(<Reports />)
    expect(container).toMatchSnapshot()
  })
})
