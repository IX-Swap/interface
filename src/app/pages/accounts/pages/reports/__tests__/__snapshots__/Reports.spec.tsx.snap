// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reports should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-1vpwcmr-MuiGrid-root"
    style="display: table;"
  >
    <div
      class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
    >
      <div
        class="makeStyles-wrapper-5 MuiBox-root css-0"
      >
        <div
          class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column makeStyles-container-3 css-1vpwcmr-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container makeStyles-header-2 css-11lq3yg-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-item css-i6prtc-MuiGrid-root"
              >
                <h2
                  class="MuiTypography-root MuiTypography-h2 css-rldqdm-MuiTypography-root"
                >
                  Reports
                </h2>
              </div>
            </div>
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <nav
                class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-128kl2r-MuiTypography-root-MuiBreadcrumbs-root"
              >
                <ol
                  class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
                />
              </nav>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-snjpuj-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-1jpm9pd"
        />
        <h4
          class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
        >
          Statements
        </h4>
        <div
          class="MuiBox-root css-1jpm9pd"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item css-89gxc5-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <div
            class="MuiBox-root css-413k17"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-1b4l2pg-MuiTypography-root"
            >
              View, manage and track your accounting reports and taxations.
            </p>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
        >
          <div
            class="makeStyles-wrapper-6 MuiBox-root css-0"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 makeStyles-filterText-7 css-1b4l2pg-MuiTypography-root"
            >
              Select Date:
            </h6>
            <div
              class="makeStyles-filtersWrapper-8 MuiBox-root css-0"
            >
              <div
                class="makeStyles-filterWrapper-9 MuiBox-root css-0"
              >
                <div
                  class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-18i04ku-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-fhk4wr-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="true"
                    for="mui-1"
                    id="mui-1-label"
                  >
                    From
                  </label>
                  <div
                    class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-jbw6h8-MuiInputBase-root-MuiOutlinedInput-root"
                    style="width: 150px;"
                  >
                    <input
                      aria-invalid="false"
                      aria-label="Choose date"
                      aria-readonly="true"
                      class="MuiOutlinedInput-input MuiInputBase-input css-17cvodc-MuiInputBase-input-MuiOutlinedInput-input"
                      id="mui-1"
                      readonly=""
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-173wfxe"
                      >
                        <span>
                          From
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
              <div
                class="makeStyles-filterWrapper-9 MuiBox-root css-0"
              >
                <div
                  class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-18i04ku-MuiFormControl-root-MuiTextField-root"
                >
                  <label
                    class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-fhk4wr-MuiFormLabel-root-MuiInputLabel-root"
                    data-shrink="true"
                    for="mui-3"
                    id="mui-3-label"
                  >
                    To
                  </label>
                  <div
                    class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-jbw6h8-MuiInputBase-root-MuiOutlinedInput-root"
                    style="width: 150px;"
                  >
                    <input
                      aria-invalid="false"
                      aria-label="Choose date"
                      aria-readonly="true"
                      class="MuiOutlinedInput-input MuiInputBase-input css-17cvodc-MuiInputBase-input-MuiOutlinedInput-input"
                      id="mui-3"
                      readonly=""
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-173wfxe"
                      >
                        <span>
                          To
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-12aw9rk"
        />
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-1vpwcmr-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded css-1nt1r1b-MuiPaper-root"
              style="background-color: inherit;"
            >
              <div
                class="MuiTableContainer-root css-rorn0c-MuiTableContainer-root"
              >
                <table
                  aria-label="table"
                  class="MuiTable-root css-7mlt34-MuiTable-root"
                  data-testid="table"
                >
                  <thead
                    class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
                    style="background-color: rgb(20, 18, 114); height: 50px; display: none;"
                  >
                    <tr
                      class="MuiTableRow-root MuiTableRow-head css-8dfetm-MuiTableRow-root"
                    >
                      <th
                        class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                        scope="col"
                      >
                        <b
                          style="color: rgb(255, 255, 255);"
                        />
                      </th>
                      <th
                        class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                        scope="col"
                      >
                        <b
                          style="color: rgb(255, 255, 255);"
                        />
                      </th>
                      <th
                        class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1efwvfh-MuiTableCell-root"
                        scope="col"
                      >
                        <b
                          style="color: rgb(255, 255, 255);"
                        />
                      </th>
                    </tr>
                  </thead>
                  <tbody
                    class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
                  >
                    <tr
                      class="MuiTableRow-root css-8dfetm-MuiTableRow-root"
                      style="background-color: rgb(247, 249, 250);"
                    >
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        Accounts Summary
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        HTML View
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-17jaypo-MuiTableCell-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 makeStyles-link-11 css-1okvd07-MuiTypography-root"
                        >
                          View Report
                        </p>
                      </td>
                    </tr>
                    <tr
                      class="MuiTableRow-root css-8dfetm-MuiTableRow-root"
                      style="background-color: rgb(248, 248, 253);"
                    >
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        Trade Confirmation
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        HTML View
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-17jaypo-MuiTableCell-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 makeStyles-link-11 css-1okvd07-MuiTypography-root"
                        >
                          View Report
                        </p>
                      </td>
                    </tr>
                    <tr
                      class="MuiTableRow-root css-8dfetm-MuiTableRow-root"
                      style="background-color: rgb(247, 249, 250);"
                    >
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        Aggregated Costs and Charges
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        HTML View
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-17jaypo-MuiTableCell-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 makeStyles-link-11 css-1okvd07-MuiTypography-root"
                        >
                          View Report
                        </p>
                      </td>
                    </tr>
                    <tr
                      class="MuiTableRow-root css-8dfetm-MuiTableRow-root"
                      style="background-color: rgb(248, 248, 253);"
                    >
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        Dividends
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignLeft MuiTableCell-sizeMedium css-17e0zod-MuiTableCell-root"
                      >
                        HTML View
                      </td>
                      <td
                        class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight MuiTableCell-sizeMedium css-17jaypo-MuiTableCell-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 makeStyles-link-11 css-1okvd07-MuiTypography-root"
                        >
                          View Report
                        </p>
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
